### DISCUSSION TITLE
表单横向布局疑问

### DISCUSSION THREAD

**Discussion by daijingfeng** [alibaba/formily#3234] [Q&A 2.x]
现在表单里按钮都是通过这种方式添加的，这样它就独占一行了，纵向排列的表单没问题。
```jsx
<FormProvider form={form}>
    <SchemaField schema={schema} />
    <FormButtonGroup>
      <Submit onSubmit={console.log}>提交</Submit>
    </FormButtonGroup>
 </FormProvider>
```
但要实现类似这样的横向布局，按钮紧跟其他表单项排列，而不是独占一行。
![image](https://user-images.githubusercontent.com/17313078/175914517-b1f81ef6-c642-4591-a6f6-096701181873.png)
现在表单设计器里，可以通过 FormLayout 组件来完成表单项的横向布局，但组件里没有按钮组件，就没法跟其他表单项一样可以通过界面来布局。

如果要实现上面这样的布局，除了在设计器里增加一个自定义的 Button 组件可供拖拽布局，又或者直接修改 schema 之外。还有其他更好的方法吗？

**Comment by ytftianwen** [comment]
https://antd.formilyjs.org/components/form-grid

可以使用form-grid进行布局
